### ISSUE TITLE
Zoom integration breaks existing Google Calendar integration and spams API calls

### ISSUE THREAD

**Issue by codedeeply** [calendso/calendso#386]
### Issue Summary
When adding a Zoom integration to a setup with an already-existing Google Calendar integration, the following issues occur:

1. The Google Calendar entries/times blocked out on the scheduling calendar get ignored, and instead each day appears fully open from the full beginning to end times set in the settings.
2. When attempting to "troubleshoot," the loading circle spins forever. During this time the Zoom API gets inundated with API requests per second, going back and forth between 200 and 429 responses ("You have reached the maximum per-second rate limit for this API. Try again later.") until the API hits a daily limit and rejects any further calls  ("You have reached the maximum daily rate limit for this API. Refer to the response header for details on when you can make another request."). Even on a 200-returned call the troubleshooting and public-facing calendar do not work correctly (troubleshooter loads forever; public-facing shows all possible meeting times as available).

Error returned in logs:

```
Error: Too Many Requests
    at handleErrorsJson (/app/.next/server/chunks/613.js:110:11)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async Promise.all (index 0)
    at async handler (/app/.next/server/pages/api/availability/[user].js:47:29)
    at async apiResolver (/app/node_modules/next/dist/next-server/server/api-utils.js:8:1)
    at async Server.handleApiRequest (/app/node_modules/next/dist/next-server/server/next-server.js:66:462)
    at async Object.fn (/app/node_modules/next/dist/next-server/server/next-server.js:58:580)
    at async Router.execute (/app/node_modules/next/dist/next-server/server/router.js:25:67)
    at async Server.run (/app/node_modules/next/dist/next-server/server/next-server.js:68:1042)
{
  code: 429,
  message: 'You have reached the maximum per-second rate limit for this API. Try again later.'
}
```

Deleting the Zoom integration fixes both issues and the program acts as expected.

### Steps to Reproduce
1. Setup a Google Calendar integration that works correctly.
2. Setup a Zoom integration using an Account Level, OAuth, non-public app. Has `meeting:write:admin` permission. Copy and paste API keys.
3. Go to "Availability" > "Launch troubleshooter"

In this setup, I would expect to be able to use the Google Calendar integration as my scheduling free/busy setup, and the Zoom integration to generate and supply meeting information for any meetings registered with Calendso. I believe this is a bug seeing that the application does not act as expected when adding a Zoom integration in this manner.

As I'm not seeing or missed documentation regarding Zoom integration setup, if I am setting it up in an incorrect manner / one not presently supported, please let me know.

Thank you!

### Technical details
* Browser version: Chrome 91 (https://www.whatsmybrowser.org/b/FO88M)
* Node.js version: 14.17.3
* This is running off `guestros/calendso-githubbuild:latest`

**Comment by joshsny** [comment]
The troubleshooting page is broken without connecting Zoom, there is an infinite render loop problem. I'll create an issue for it now and submit a PR.

**Comment by Nico-J** [comment]
Somehow, the PR #429 is empty and closed without being merged. But @joshsny has fixed it in another PR #434 

I've made a tiny suggestion; after that this should be fixed. :) Thank you very much for your contribution, Joshua!
